ibtk/src/math/HierarchyMathOps.cpp

```diff
@@ -1930,6 +1930,102 @@ HierarchyMathOps::interp(const int dst_idx,
     return;
 } // interp
 
+void
+HierarchyMathOps::interp(const int dst_idx,
+                         const Pointer<NodeVariable<NDIM, double> > /*dst_var*/,
+                         const int src_idx,
+                         const Pointer<FaceVariable<NDIM, double> > /*src_var*/,
+                         const Pointer<HierarchyGhostCellInterpolation> src_ghost_fill,
+                         const double src_ghost_fill_time,
+                         const bool src_cf_bdry_synch)
```

Comment:
I should have noticed this earlier, but for this `interp` function, it probably is necessary also to have an option for `dst_cf_bdry_synch`. Currently I don't think that the nodal representation will generally have consistent values at coarse-fine interfaces.